extensions/amp-video/amp-video.md

```diff
@@ -103,6 +101,12 @@ If present, will automatically loop the video back to the start upon reaching th
 
 This element includes [common attributes](https://www.ampproject.org/docs/reference/common_attributes) extended to AMP components.
 
+## Click-to-Play overlay
+Click-to-play is a common UX for video players on the web. `amp-video` supports
+`play` standard AMP action which allows us to implement this pattern overlays with ease.
```

Comment:
Rewrite as:

the standard `play` AMP action, allowing you to implement click-to-play easily.